src/ext/java/org/opentripplanner/ext/siri/updater/azure/SiriAzureETUpdater.java

```diff
@@ -174,4 +181,25 @@ private List<EstimatedTimetableDeliveryStructure> getUpdates(String message, Str
 
     return siri.getServiceDelivery().getEstimatedTimetableDeliveries();
   }
+
+  private void logErrors(UpdateResult updateResult) {
+    if (updateResult.failed() == 0) {
+      return;
+    }
+    var errorIndex = updateResult.failures();
```

Comment:
This is pretty much a copy and paste job from https://github.com/opentripplanner/OpenTripPlanner/blob/2e3393766ba03d45eb2a965decad5a7a2bafde96/src/main/java/org/opentripplanner/updater/spi/ResultLogger.java#L32-L45

Can you introduce a reusable method into `ResultLogger` and use it here?

Reply:
Sure! I'll do that